app/java/src/main/resources/templates/error-unspecified-processed-data-bucket.html

```diff
@@ -0,0 +1,17 @@
+<!--
```

Comment:
FWIW, I did investigate the engineering required to remove this and other html headers from rendered pages, but given unit and infra testing successfully search for substrings, and many other samples probably send html comments full of license headers, I didn't log this as a separate nit (also applies to existing templates, not just this file)